[09:03:23] [Info] こんにちは！ Booting Zettlr at Mon Oct 02 2023 09:03:23 GMT-0300 (Brasilia Standard Time).
[09:03:23] [Info] Registering custom protocol safe-file
[09:03:23] [Warning] [Assets Provider] Found outdated filter links.lua; copying ...
[09:03:23] [Warning] [Assets Provider] Found outdated filter tags.lua; copying ...
[09:03:23] [Info] [Citeproc Provider] Loading CSL locale file at /usr/lib/zettlr/resources/app.asar/.webpack/main/assets/csl-locales/locales-en-US.xml ...
[09:03:23] [Info] [Update Provider] Checking for application updates ...
[09:03:23] [Info] [Updater] Checking https://zettlr.com/api/releases/latest for updates ...
[09:03:23] [Info] [FSAL] Loaded all files and workspaces in 0.08 seconds
[09:03:23] [Info] [Window Manager] Window Manager started.
[09:03:23] [Info] [Document Manager] Restored 1 open windows.
[09:03:23] [Info] [AppServiceContainer] Boot successful!
[09:03:23] [Info] [Application] Added /usr/lib/zettlr/resources to PATH.
[09:03:23] [Info] [Application] The bundled pandoc executable is now in PATH. If you do not want to use the bundled pandoc, uncheck the corresponding setting and reboot the app.
[09:03:24] [Error] [Update Provider] UNABLE_TO_GET_ISSUER_CERT_LOCALLY: Could not check for updates. RequestError: unable to get local issuer certificate
[09:03:27] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:03:27] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:03:27] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:03:27] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:03:30] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.solicitacao-reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.solicitacao-reajuste-preco
[09:03:30] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.reajuste-preco
[09:03:47] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[09:03:54] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:03:56] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/pricing/mip.md
[09:03:56] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/pricing/mip.md
[09:03:56] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/pricing/mip.md saved.
[09:03:57] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:10:05] [Error] [Application] Could not fetch link preview for URL https://mip-dev.cluster-cu62elblagyw.sa-east-1.rds.amazonaws.com: connect ETIMEDOUT 172.24.137.181:443
[09:36:09] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:36:12] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:12] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:12] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[09:36:20] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:36:22] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:22] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:22] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[09:36:24] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:36:38] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:36:41] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:41] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:36:41] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[09:44:54] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[09:44:57] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:44:57] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[09:44:57] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[11:44:38] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[11:44:48] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[11:45:00] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[11:45:03] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[11:45:03] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[11:45:03] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[13:10:47] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:10:58] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:11:00] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:00] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:00] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[13:11:22] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:11:25] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:11:27] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:27] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:27] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[13:11:36] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:11:38] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:38] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:38] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[13:11:43] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:11:45] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:45] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[13:11:45] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[13:15:45] [Error] [Application] Could not fetch link preview for URL https://seller.autoglass.com.br/inscricao.: Response code 404 (Not Found)
[13:15:47] [Error] [Application] Could not fetch link preview for URL https://autoglass.produtos.importacao: getaddrinfo ENOTFOUND autoglass.produtos.importacao
[13:15:48] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:15:48] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:15:48] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:15:48] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:15:59] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:15:59] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:15:59] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:15:59] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:16:03] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:04] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:04] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:16:04] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:16:04] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:16:04] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:16:08] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:16:08] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:16:10] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:16:12] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/seller/seller.md
[13:16:12] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/seller/seller.md
[13:16:12] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/seller/seller.md saved.
[13:16:15] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:15] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:20] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:24] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:16:27] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:17:36] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:17:36] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:17:36] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[13:17:36] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[13:17:42] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:17:49] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[13:17:51] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:17:52] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[13:17:52] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md
[13:17:52] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md
[13:17:52] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md saved.
[13:17:54] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md
[13:17:54] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md
[13:17:54] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/timeLog/timeLog.md saved.
[13:17:56] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[16:36:39] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[16:36:44] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[16:36:46] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[16:36:46] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[16:36:46] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[16:36:51] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[16:36:57] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[16:37:00] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[16:37:00] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md
[16:37:00] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/Tarefas.md saved.
[17:16:31] [Error] [Application] Could not fetch link preview for URL https://seller.autoglass.com.br/inscricao.: Response code 404 (Not Found)
[17:16:32] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:18:40] [Error] [Application] Could not fetch link preview for URL https://seller.autoglass.com.br/inscricao.: Response code 404 (Not Found)
[17:18:42] [Error] [Application] Could not fetch link preview for URL https://autoglass.produtos.importacao: getaddrinfo ENOTFOUND autoglass.produtos.importacao
[17:18:43] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[17:18:43] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[17:18:43] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos.importacao: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos.importacao
[17:18:46] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.seller.produtos: getaddrinfo ENOTFOUND autoglass.comercio.seller.produtos
[17:18:58] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:18:59] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:19:41] [Error] [Application] Could not fetch link preview for URL https://bitbucket.org/mgvidros/diagramas-arq-referencia;: Response code 404 (Not Found)
[17:19:44] [Error] [Application] Could not fetch link preview for URL https://autoglass.com.br): getaddrinfo ENOTFOUND autoglass.com.br)
[17:19:45] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:19:45] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.vendas.venda: getaddrinfo ENOTFOUND autoglass.comercio.vendas.venda
[17:19:52] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.vendas.venda: getaddrinfo ENOTFOUND autoglass.comercio.vendas.venda
[17:19:58] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.vendas.venda: getaddrinfo ENOTFOUND autoglass.comercio.vendas.venda
[17:20:01] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:02] [Error] [Application] Could not fetch link preview for URL https://pkc-n00kk.us-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:07] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:09] [Error] [Application] Could not fetch link preview for URL https://pkc-n00kk.us-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:15] [Error] [Application] Could not fetch link preview for URL https://pkc-n00kk.us-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:18] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:20:39] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:20:47] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:20:50] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:21:03] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:21:08] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:21:19] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:21:22] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:21:22] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:21:22] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md saved.
[17:21:27] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:21:29] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:21:29] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:21:29] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md saved.
[17:23:21] [Info] [DocumentManager] Processing change for /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:23:21] [Info] [WATCHDOG] Emitting event: change:/home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md
[17:23:21] [Info] [DocumentManager] File /home/fabioscalzer/projetos/documentação/documentacao/comercio/tracking/tracking.md saved.
[17:23:22] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:24:05] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:24:06] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.reajuste-preco
[17:24:06] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.solicitacao-reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.solicitacao-reajuste-preco
[17:24:06] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.solicitacao-reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.solicitacao-reajuste-preco
[17:24:06] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.reajuste-preco
[17:24:07] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.solicitacao-reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.solicitacao-reajuste-preco
[17:24:07] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.reajuste-preco
[17:24:07] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.reajuste-preco
[17:24:07] [Error] [Application] Could not fetch link preview for URL https://autoglass.comercio.mip.integracao.solicitacao-reajuste-preco: getaddrinfo ENOTFOUND autoglass.comercio.mip.integracao.solicitacao-reajuste-preco
[17:24:21] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:24:23] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:24:23] [Error] [Application] Could not fetch link preview for URL https://pkc-4v5zz.sa-east-1.aws.confluent.cloud:9092: socket hang up
[17:24:24] [Error] [Application] Could not fetch link preview for URL https://pkc-n00kk.us-east-1.aws.confluent.cloud:9092: socket hang up
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:51] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-aplicacao.fifo: Response code 404 (Not Found)
[17:28:52] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:52] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-aplicacao.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-promocao-produto-importacao-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:53] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-aplicacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/754529021164/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/069216277836/comercio-reajuste-oportunidade-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-aplicacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-temporario.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-agregado-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-aplicacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-promocao-produto-importacao.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-temporario.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-aplicacao-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-alteracao-preco-temporario-erro.fifo: Response code 404 (Not Found)
[17:28:54] [Error] [Application] Error running LanguageTool: unable to get local issuer certificate | Native Error: RequestError; unable to get local issuer certificate Stack Trace: RequestError: unable to get local issuer certificate -->     at ClientRequest.<anonymous> (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:835421) -->     at Object.onceWrapper (node:events:628:26) -->     at ClientRequest.emit (node:events:525:35) -->     at e.emit (/usr/lib/zettlr/resources/app.asar/.webpack/main/index.js:2:11466) -->     at TLSSocket.socketErrorListener (node:_http_client:502:9) -->     at TLSSocket.emit (node:events:513:28) -->     at emitErrorNT (node:internal/streams/destroy:151:8) -->     at emitErrorCloseNT (node:internal/streams/destroy:116:3) -->     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) -->     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34) -->     at TLSSocket.emit (node:events:513:28) -->     at TLSSocket._finishInit (node:_tls_wrap:959:8) -->     at ssl.onhandshakedone (node:_tls_wrap:743:12)
[17:28:54] [Error] [Application] Could not fetch link preview for URL https://sa-east-1.queue.amazonaws.com/973580931382/comercio-reajuste-oportunidade-temporario.fifo: Response code 404 (Not Found)
[17:30:39] [Error] [Application] Could not fetch link preview for URL https://mip-dev.cluster-cu62elblagyw.sa-east-1.rds.amazonaws.com: connect ETIMEDOUT 172.24.137.181:443
[18:00:11] [Info] さようなら！ Shutting down at Mon Oct 02 2023 18:00:11 GMT-0300 (Brasilia Standard Time)
[18:00:11] [Info] Application uptime was: 8 hours, 56 minutes, and 49.389 seconds.
[18:00:11] [Info] [FSAL Cache] Cleaned up cache: Removed 0 remnants.
